torch_frame/data/dataset.py

```diff
@@ -551,10 +551,13 @@ def get_split(self, split: str) -> 'Dataset':
 
     def split(self) -> Tuple['Dataset', 'Dataset', 'Dataset']:
         r"""Splits the dataset into training, validation and test splits."""
+        train_split = self.get_split('train')
+        val_split = self.get_split('val')
+        test_split = self.get_split('test')
         return (
-            self.get_split('train'),
-            self.get_split('val'),
-            self.get_split('test'),
+            train_split,
+            val_split,
+            test_split,
```

Comment:
Can you remove this change as it seems irrelevant to this pr?

Reply:
Sure